ansible.el

```diff
@@ -121,6 +126,24 @@
        (f-files ansible::root-path (lambda (file) (s-matches? ".yml" (f-long file))) t))
     nil))
 
+(defun ansible::vault (mode)
```

Comment:
I like "divide `encrypt/decrypt str` func and `buffer change` func".

because, I think `ansible::vault()` func name is genenal name.

```elisp
(defun ansible::vault (str mode)
 ...
   encrypt-or-decrypt-str)

(defun ansible::vault-buffer (mode)
   (let (str (buffer-substring-no-properties (point-min) (point-max)))
       (delete-region (point-min) (point-max))
       (insert (ansible::vault mode str))))
```